src/providers/flagsView.ts

```diff
@@ -470,17 +466,12 @@ export class FlagNode extends vscode.TreeItem {
 		this.conditionalIcon(ctx, this.contextValue, this.label);
 	}
 
-	private conditionalIcon(ctx: vscode.ExtensionContext, contextValue: string, label: string) {
+	private conditionalIcon(ctx: vscode.ExtensionContext, contextValue: string, label: string, enabled?: boolean) {
```

Comment:
This new param appears to be unused in `conditionalIcon`

Reply:
without it the Interface does not work, it says the signatures do not match. I'm not sure how to approach it better.